src/FileSystem-Disk/DiskStore.class.st

```diff
@@ -99,6 +139,31 @@ DiskStore class >> startUp: resuming [
 	DefaultWorkingDirectory := self defaultWorkingDirectory.
 ]
 
+{ #category : #public }
+DiskStore class >> wantsReadOnly [
+
+	| arguments | 
+	
+	arguments := Array
+		streamContents: [ :str | 
+			| arg i |
+			i := 0.
+			[ i > 998 or: [ (arg := Smalltalk argumentAt: i) isNil ] ]
+				whileFalse: [ 
+					str nextPut: arg.
+					i := i + 1 ] ].
```

Comment:
I wonder we do not have nice API to access console options?
It's definitely should be extracted to some right place